tests/resources/iframeAPITest.html

```diff
@@ -0,0 +1,118 @@
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="content-type" content="text/html;charset=utf-8">
+    <title>iframe API test</title>
+</head>
+<body>
+<script>
+    const blacklist = [ '__proto__', 'constructor', 'prototype' ];
```

Comment:
This is copied from somewhere, right? Maybe put a comment about where it comes from?

Reply:
From jitsi-meet-torture. https://github.com/jitsi/jitsi-meet-torture/blob/master/src/test/resources/files/iframeAPITest.html

Reply:
To put in comments that it comes from jitsi-meet-torture?

Reply:
Yes please.